DYDX-Bots/Frozen-monitor/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "dydx-frozen-monitor",
+  "version": "0.0.1",
+  "description": "Bot that detects frozen state changes in dydx perpetual contract.",
+  "chainIds": [
+    1
+  ],
```

Comment:
We need to included `"repository"` in `package.json`.

Reply:
Yes, I believe we need to do this after merging to the main branch.